.github/workflows/build_docker.yml

```diff
@@ -31,18 +31,19 @@ jobs:
         with:
           path: "./"
         id: package-version
+        if: runner.os != 'Windows'
 
       - name: Get image tag
         id: get-image-tag
         run: |
           if ${{ github.event_name == 'workflow_dispatch' }} ; then
             if ${{ inputs.image_tag == 'release'}}; then
-              echo "image_tag=${{ steps.package-version.outputs.version }}" >> $GITHUB_OUTPUT
+              echo "image_tag=${{ steps.package-version.outputs.version }}" >> $GITHUB_ENV
```

Comment:
The `GITHUB_OUTPUT` env variable should be left alone here and elsewhere in this workflow - this is the replacement for the old `"::set-output` command and should not trigger warnings.

The deprecation warnings are coming from the third-party actions `tyankatsu0105/read-package-version-actions@v1` ([on L30](https://github.com/aws/graph-explorer/blob/main/.github/workflows/build_docker.yml#L30)) and `aws-actions/configure-aws-credentials@v1` ([on L49](https://github.com/aws/graph-explorer/blob/main/.github/workflows/build_docker.yml#L49)). 

We should be able to upgrade the latter to  `aws-actions/configure-aws-credentials@v2` without issue (see [here](`aws-actions/configure-aws-credentials@v1)). The first action for reading the JS package version will need a replacement library, as the author has not updated it in several years.

Reply:
@michaelnchin 
I went ahead a reverted the changes made to the Get image tag step, updated those actions, and replaced the package version with what looks more promising action.

Reply:
Awesome, thank you for the updates @cubeddu 👍